platforms/gcp-dataproc-classical-airgapped/spark_healthcare_nlp_classical.py

```
@@ -0,0 +1,77 @@
+
+    return builder.getOrCreate()
+
+
+print("**********************Starting Spark Session...***************************")
+spark = start()
+print("**********************Spark session ready! ***************************")
+
+
+
+print("******************** Starting building the spark NLP Healthcare pipeline... ********************* ")
+document_assembler = DocumentAssembler()\
```

Ruff (0.11.9)

`DocumentAssembler` may be undefined, or defined from star imports

(F405)